.lintstagedrc.json

```diff
@@ -0,0 +1,3 @@
+{
+    "*.js": ["prettier --write", "eslint --fix"]
```

Comment:
File can be any format, for example json or ejs, cjs

Reply:
> File can be any format, for example json or ejs, cjs

I can't see this file extension in this project. Sortt

Reply:
I meant config file like .json